### ISSUE TITLE
Intermediate romanizer directly before "Syllables: clear" clears the syllables before printing the romanization

### ISSUE THREAD

**Issue by lane-eden** [def-gthill/lexurgy#47]
For example:

```
...

Romanizer-after-changes:
 unchanged

Syllables:
 clear
```

Expected output:
`pika   => ˈpi.ka    => ˈɸi.ɣa => ɸiɣa`

Actual output:
`pika   => ˈpi.ka    => ɸiɣa => ɸiɣa`

Workaround: If you insert a null rule, then the output is the expected output above:

```
...

Romanizer-after-changes:
 unchanged

null-rule:
 unchanged

Syllables:
 clear
```

**Comment by def-gthill** [comment]
Thanks for reporting! I had the syllabification rules and the intermediate romanizers between any two rules all kind of thrown together into a bag, and then the syllabification rules would always run before the romanizers, even if they were declared in a different order. This is fixed in 2c6a5c2a36414c3e49140fb64d1972453da1dee0 and will be included in the next release.

**Comment by lane-eden** [comment]
Awesome! Thank you!
